sound/soc/sof/sof-audio.c

```diff
@@ -433,6 +433,33 @@ struct snd_sof_dai *snd_sof_find_dai(struct snd_soc_component *scomp,
 	return NULL;
 }
 
+/*
+ * Helper to get SSP MCLK from a pcm_runtime.
+ * Return 0 if not exist.
+ */
+int snd_sof_dai_get_ssp_mclk(struct snd_soc_pcm_runtime *rtd)
+{
+	struct snd_soc_component *component =
+		snd_soc_rtdcom_lookup(rtd, SOF_AUDIO_PCM_DRV_NAME);
```

Comment:
So this looks to be a generic function that retrieve the DAI MCLK? Would it be possible to remove SSP from its name?

Maybe ``snd_sof_dai_get_mclk``. IMX DAIs have different names (e.g SAI, ESAI).